type_description_interfaces/README.md

```diff
@@ -0,0 +1,22 @@
+# type_description_interfaces
+
+A package containing message and service definitions for describing and communicating descriptions of other types.
+
+## Messages (.msg)
+
+* [Field](msg/Field.msg): Represents a single field in a ROS Interface Type
+* [FieldType](msg/FieldType.msg): Represents the type of a single field in a ROS Interface Type
+* [IndividualTypeDescription](msg/IndividualTypeDescription.msg): Represents a single ROS Message Type by itself
+* [TypeDescription](msg/TypeDescription.msg): Represents a single ROS Message Type along with all referenced types
+
+## Quality Declaration
+
+This package claims to be in the **Quality Level 1** category, see the [Quality Declaration](QUALITY_DECLARATION.md) for more details.
+
+## TODO
+
+* How to store default values, should store default values? Current storing as string, but will impact hashing.
```

Comment:
Maybe not relevant anymore since we've decided TypeDescription has to be serialized for hashing, so default values can be skipped in that process.

Reply:
Right, since you're now ignoring parts of this for the purposes of hashing and we have the default values in already, we can just remove this todo.